pkgs/applications/science/misc/openmodelica/omshell/default.nix

```diff
@@ -0,0 +1,34 @@
+{lib, qttools, qmake, qtwebkit, wrapQtAppsHook, readline, openmodelica, mkOpenModelicaDerivation}:
+
+with openmodelica;
```

Comment:
Please only apply with where it is required and not over the entire file.

Reply:
OK!